src/test/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/CpxSVInferenceTestUtils.java

```diff
@@ -151,7 +151,7 @@ private static PreprocessedAndAnalysisReadyContigWithExpectedResults buildForCon
                 .id("CPX_chr2:4452298-4452406")
                 .attribute(VCFConstants.END_KEY, 4452406)
                 .attribute(SVTYPE, GATKSVVCFConstants.CPX_SV_SYB_ALT_ALLELE_STR)
-                .attribute(SVLEN, 109)
+                .attribute(SVLEN, 783)
```

Comment:
this would be gone in this PR, as explained in the commit message. 
Basically, I'm changing the SVLEN field to follow its technical definition 
in VCF spec, i.e. for precise variants the difference between reference and alt alleles.